tests/end_to_end/features/steps/client_api.py

```diff
@@ -0,0 +1,35 @@
+from behave import then
+
+
+@then('I should get the columns for the data source from "{provider}"')
+def step_impl(context, provider):
+    context.client.apply()
+    context.cols = context.client.columns(context.transactions)
+    print("\n\n")
+    print("ACTUAL COLUMNS: ", context.cols)
+    print("\n\n")
```

Comment:
do we need these debug prints still?

Reply:
Oops. Thought I pulled these out. Thank you!